pkg/logflags/logflags.go

```diff
@@ -29,17 +29,24 @@ var minidump = false
 
 var logOut io.WriteCloser
 
-func makeLogger(flag bool, fields logrus.Fields) *logrus.Entry {
-	logger := logrus.New().WithFields(fields)
-	logger.Logger.Formatter = &textFormatter{}
+func makeFlaggableLogger(flag bool, fields Fields) Logger {
```

Comment:
No reason to split this function in to two functions when you only call makeFlaggableLogger anyway.

Reply:
This is caused by [`pkg/logflags/logflags.go:163`](https://github.com/go-delve/delve/pull/3257/files/ccda6c241cf732d8ce033eb443f31beead8da3d7#diff-f89e32a213ab96ab1585632446e21aef2976688c4ac06c461d6ceb95bcc2903cR163):
```golang
logger := RPCLoggerWithLevel(logrus.WarnLevel)
logger.Warnln("Listening for remote connections (connections are not authenticated nor encrypted)")
```

In this case the initial code forced that this Warning should always be displayed, although if tagged or not. With this I tried to preserve this behavior.

Reply:
We can just pass true as the flag.

Reply:
Ok.